packages/dataviews/src/bulk-actions.tsx

```diff
@@ -248,7 +249,11 @@ export default function BulkActions< Item extends AnyItem >( {
 						disabled={ areAllSelected }
 						hideOnClick={ false }
 						onClick={ () => {
-							onSelectionChange( selectableItems );
+							onSelectionChange(
```

Comment:
Before, this used an `Item[]`, now it uses `string[]`. Is this problematic?

Reply:
Consumers are expecting an `Item[]` ([pages](https://github.com/WordPress/gutenberg/blob/b86375d38920c3938b3ff6ecd6a1aa24af5ab7a8/packages/edit-site/src/components/page-pages/index.js#L227), [templates](https://github.com/WordPress/gutenberg/blob/b86375d38920c3938b3ff6ecd6a1aa24af5ab7a8/packages/edit-site/src/components/page-templates/index.js#L240)), so they need to be updated.

Reply:
What should `onSelectionChange` offer to consumers, id (`string[]`) or items (`Item[]`)? 

I'm inclined to work with `Item[]`.

It enables consumers to immediately act on any needs they have instead of having to look the item up based on id. For example, upon click, the consumer may want to display extra info about the item in a panel outside of dataviews — if they have the item they already have they info, otherwise, they'd have to look it up based on id, first.

Reply:
This is not the public API

Reply:
`onSelectionChange` still offers `Item[]` to consumers, I didn't change that public API at all. This is just setting internal state, which was already storing only the IDs. Instead of passing Items around internally it now just passes IDs around.

Reply:
Oh, I had just glanced at the code, sorry. Gave it a second/deeper look now and I see the changes 👍